.github/workflows/ci.yml

```diff
@@ -33,18 +33,16 @@ jobs:
       - name: Install dependencies
         shell: bash -l {0}
         run: |
-          conda install -c conda-forge pip botocore aiobotocore==2.1.0 "aiohttp<=4" "moto>=2.0,<3" pytest flake8 black -y
+          pip install -r requirements.txt -r test_requirements.txt
           pip install git+https://github.com/fsspec/filesystem_spec --no-deps
-          conda list
```

Comment:
It's always useful to list the versions that are actually being used in the test. Pip is fine for this.